home/screens/AccountModal/LoggedInAccountView.tsx

```diff
@@ -81,23 +81,34 @@ export function LoggedInAccountView({ accounts }: Props) {
                     </View>
                   )}
                   <Spacer.Horizontal size="small" />
-                  <View>
+                  <View flex="1">
                     {account.owner ? (
                       <>
                         {account.owner.fullName ? (
                           <>
-                            <Text type="InterBold">{account.owner.fullName}</Text>
+                            <Text type="InterBold" style={{ paddingRight: 16 }} numberOfLines={1}>
```

Comment:
```suggestion
                            <Text type="InterBold" style={{ paddingRight: spacing[4] }} numberOfLines={1}>
```

Just a small suggestion to switch to `spacing` property form styleguide, not required at all, can be addressed in the future. 😉